BraveWallet/Extensions/WalletTxServiceExtensions.swift

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import BraveCore
+
+extension BraveWalletTxService {
+  
+  // Fetches all pending transactions for all given keyrings
+  func pendingTransactions(
+    for keyrings: [BraveWallet.KeyringInfo]
+  ) async -> [BraveWallet.TransactionInfo] {
+    var pendingTransactions: [BraveWallet.TransactionInfo] = []
+    pendingTransactions = await withTaskGroup(
```

Comment:
This can just be `return await withTaskGroup(...`

Reply:
fixed in [1aaf290](https://github.com/brave/brave-ios/pull/5746/commits/1aaf290ebb80a10973b88d03bca9c0ed1647cf34)